tests/unit/api/metadata/test_v0.py

```diff
@@ -20,11 +20,10 @@ def setUp(self) -> None:
                 {
                     'cluster': 'test_cluster',
                     'database': 'test_db',
-                    'key': 'test_db://test_cluster.test_schema/test_table',
                     'schema': 'test_schema',
-                    'table_description': 'This is a test',
-                    'table_name': 'test_table',
-                    'type': 'table',
+                    'description': 'This is a test',
+                    'name': 'test_table',
+                    'key': 'test_db://test_cluster.test_schema/test_table',
```

Comment:
I think this should have both `type` and `key`

Reply:
ah, forgot to add the key to expected parsed user resources.